test/functional/test_framework/authproxy.py

```diff
@@ -139,17 +139,24 @@ def _request(self, method, path, postdata):
             return self._get_response()
 
     def __call__(self, *args, **argsn):
-        AuthServiceProxy.__id_count += 1
+        # check for a possible wallet endpoint
+        urlAdd = ""
+        method = self._service_name
+        elements = self._service_name.split(".")
+        if len(elements) > 1:
+            urlAdd = "/v1/wallet/" + '.'.join(elements[:-1])
```

Comment:
Thread https://github.com/bitcoin/bitcoin/pull/10650#discussion_r126976236

Test framework comment not (yet) addressed.